lib/Util/TextFormat.php

```diff
@@ -18,9 +18,25 @@ public function indent(string $string, int $level = 0)
     {
         $lines = explode(PHP_EOL, $string);
         $lines = array_map(function ($line) use ($level) {
+            if (!$line) {
+                return $line;
+            }
             return str_repeat($this->indentation, $level) . $line;
         }, $lines);
 
         return implode(PHP_EOL, $lines);
     }
+
+    public static function indentationRemove(string $text): string
+    {
+        $text = preg_replace("/\R +/m", "\n", $text);
+        $text = preg_replace("/^ +/m", "", $text);
```

Comment:
Pretty sure one of these can be removed and the tests still pass...

Reply:
The first one doesn't seem useful, also it replace the carriage return which could be a bad thing.